app/models/casa_org.rb

```diff
@@ -85,11 +86,17 @@ def set_slug
     self.slug = name.parameterize
   end
 
-  def generate_contact_types_and_hearing_types
+  def generate_defaults
     ActiveRecord::Base.transaction do
       ContactTypeGroup.generate_for_org!(self)
       HearingType.generate_for_org!(self)
+      ContactTopics.generate_for_org!(self)
     end
+    # FIXME: is this needed? do we care about the case where the defaults
+    # are invalid?
+  rescue ActiveRecord::RecordInvalid => e
```

Comment:
I'm wasn't too sure how to address the situation where the defaults are invalid? Should we pretend it can't happen?